search/includes/classes/class-health.php

```diff
@@ -12,6 +12,8 @@
 class Health {
 	const CONTENT_VALIDATION_BATCH_SIZE    = 500;
 	const CONTENT_VALIDATION_MAX_DIFF_SIZE = 1000;
+	const CONTENT_VALIDATION_LOCK_NAME = 'vip_search_content_validation_ongoing';
```

Comment:
I think this should be `vip_search_content_validation_lock` since that's what it is.

What do you think?